src/auto-trace.helper.js

```diff
@@ -84,14 +73,22 @@ export function appendExtraContext(error, extraContext){
 	return errOut
 }
 
+export function createError(framesToRemove = 1){
+	const error = new Error();
+	//We remove an extra frame since this function will create a frame as well
+	const newStack = error.stack.split('\n');
+	newStack.splice(1,framesToRemove+1);
+	error.stack = newStack.join('\n');
+	return error;
+}
+
 /**
- * Removes auto-trace from the error stack
+ * Adds error message to the error stack string
  * @param  {Error} err
  * @return {Error}
  */
-export function removeAutoTraceFromErrorStack(err){
+function addErrorMessageToStack(err){
 	if(err instanceof Error && typeof err.stack === "string"){
-		err.stack = err.stack.replace(/\n.*(?:yncStacktrace|wrapObjectWithError) ?\(.*auto-trace.*/g,'');
```

Comment:
no longer regex removes `auto-trace` from stack. Instead createError() is used to create errors that do not contain auto trace.